scripts/lib/mlkem_test.py

```diff
@@ -355,7 +408,8 @@ def _func(opt: bool):
             exit(1)
 
     def _run_nistkat(self, opt: bool):
-        def expect_proc(scheme: SCHEME, actual: str) -> tuple[bool, str]:
+        def expect_proc(scheme: SCHEME, actual: str):
```

Comment:
In other places you removed the `:str` type information. 
Should that be removed in all places?

Reply:
Fixed (hopefully)